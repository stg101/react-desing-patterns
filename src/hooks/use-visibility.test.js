import React from "react";
import { render } from "@testing-library/react";

import useVisibility from "./use-visibility";

test("Use visibility hook", () => {
  function Component() {
    const visibility = useVisibility();
    return <div>The visibility page is: {visibility.toString()}</div>;
  }
  const { asFragment } = render(<Component />);

  expect(asFragment()).toMatchSnapshot();
});
