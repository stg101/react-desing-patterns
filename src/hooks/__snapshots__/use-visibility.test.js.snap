// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Use visibility hook 1`] = `
<DocumentFragment>
  <div>
    The visibility page: true
  </div>
</DocumentFragment>
`;
